src/cli.rs

```diff
@@ -62,7 +62,7 @@ pub struct Rover {
     log_level: Option<Level>,
 
     #[clap(flatten)]
-    output_strategy: OutputStrategy,
```

Comment:
renamed this to `output_opts` so it's clear that this is the type coming from CLI config